js/render/core/renderer.js

```diff
@@ -762,6 +763,19 @@ export class Renderer {
               gl.uniformMatrix4fv(program.uniform.LEFT_VIEW_MATRIX, false, views[0].viewMatrix);
               gl.uniformMatrix4fv(program.uniform.RIGHT_PROJECTION_MATRIX, false, views[1].projectionMatrix);
               gl.uniformMatrix4fv(program.uniform.RIGHT_VIEW_MATRIX, false, views[1].viewMatrix);
+
+              // for older browser that don't support projectionMatrix and transform on the depth data
```

Comment:
NIT: Also for view-aligned depth data?

Reply:
I don't think so. The way the spec is written, it will return the view/projection that is in the view

Reply:
No, it's identity in the spec at the moment (and how I had just implemented it/sent a WPT for): https://immersive-web.github.io/depth-sensing/#ref-for-xrdepthinformation-sensor

Reply:
That conflicts with the algoritm for `getDepthInfomation`.
See https://immersive-web.github.io/depth-sensing/#create-a-webgl-depth-information-instance

Reply:
It doesn't look like we have any other description for `projectionMatrix`; I do agree that there is a conflict with that algorithm. I don't have a problem with a spec text change to match what you're doing here (e.g. change my link to say that they must be equal to view's projectionMatrix and transform in that case), as I think it's a more elegant solution, since they can't be nullable with the mixin (at least, to my knowledge), and then we just have one consistent spot of "This is the data you should use" going forward, and the `matchDepthView` is still an optimization hint.

I can make it or you can if you'd prefer. Are there any other spots you think should be cleaned up with that change then?

Reply:
I'm neutral. If you're busy, I'm happy to do it.